frontend/app/controllers/spree/checkout_controller.rb

```diff
@@ -150,7 +150,8 @@ def ensure_order_not_completed
 
     def ensure_sufficient_stock_lines
       if @order.insufficient_stock_lines.present?
-        flash[:error] = Spree.t(:inventory_error_flash_for_insufficient_quantity)
+        out_of_stock_items = @order.insufficient_stock_lines.map { |l| "\"#{l.name}\"" }.join(', ')
```

Comment:
Will do. I'm actually wrapping each name in `"` to distinguish it from the rest of the message.